nfc.py

```diff
@@ -2,7 +2,9 @@
 
 import logging
 import sys
-import usb.core
+import pkg_resources
```

Comment:
This is making "setuptools" package required! We do not want to have such dependency!